coalahtml/_coalahtml/tests/specs/directives/BsNavActiveDirectiveSpec.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+describe('BsNavActive Directive', function() {
```

Comment:
Im confused here. Not sure what's happening.
This was a commit i've already added.
On checking it out, this branch has 36 commits while master has 46, so you've squashed 9 commits of master into this 1 commit ?